src/pyforest/user_specific_imports.py

```diff
@@ -60,9 +60,25 @@ def _get_imports_from_user_settings(user_imports_path) -> list:
 
 
 def _assign_imports_to_globals(import_statements: list, globals_) -> None:
-    symbols = [import_statement.split()[-1] for import_statement in import_statements]
-
-    for symbol, import_statement in zip(symbols, import_statements):
+    symbols = []; new_import_statements = []
+    
+    for import_statement in import_statements:
+        def process(statement):
+            symbols.append(statement.split()[-1])
+            new_import_statements.append(statement)
+
+        if "," not in import_statement:
```

Comment:
Thanks for reply!

I didn't change any the original behaviors.

I only put line 70. to judge whether the import statement include multiple imports.

if "," not in import_statement,
"from keras.layers import LSTM" still behaves as the same as original code.

if "," is in import_statement, "from keras.layers import LSTM, Dropout, Dense",
it will become
"from keras.layers import LSTM"
"from keras.layers import Dropout"
"from keras.layers import Dense".
Then, the following behaviors are still the same as the original